bcda/api.go

```diff
@@ -331,6 +338,11 @@ func jobStatus(w http.ResponseWriter, r *http.Request) {
 		}
 
 		w.WriteHeader(http.StatusOK)
+	case "Archived":
+		fallthrough
+	case "Expired":
+		w.Header().Set("Expires", job.CreatedAt.Add(GetJobTimeout()).String())
+		responseutils.WriteError(&fhirmodels.OperationOutcome{}, w, http.StatusGone)
```

Comment:
The `Expires` header and 410 HTTP status should convey the message that the job expired, but I'm wondering if we should still use `CreateOpOutcome()` here to put some details in the OperationOutcome. Maybe a separate ticket?